pkgs/top-level/all-packages.nix

```diff
@@ -33015,6 +33015,8 @@ with pkgs;
   }) k3s_1_26 k3s_1_27 k3s_1_28;
   k3s = k3s_1_27;
 
+  k3s_1_29 = callPackage ../applications/networking/cluster/k3s { };
```

Comment:
It seems referencing it like this will build all packages in the file, running a build/test today on a fixed line to make sure it behaves like expected.

Reply:
My bad. Solution seems to be to just use `inherit` then.

Reply:
That's what I changed to, was just waiting on the test runs to finish before pushing.